tests/unit/commands/deploy/test_guided_config.py

```diff
@@ -3,7 +3,9 @@
 from unittest.mock import patch
 
 import click
-from click.globals import _local
+
+# TODO: importing private module property, to investigate alternative for stability
+from click.globals import _local  # type: ignore
```

Comment:
Why does mypy care about this? 🤔 Wouldn't setting types on the "private" method work?

Reply:
mypy seems to be unhappy about any private imports.